lib/main.ts

```diff
@@ -305,6 +312,22 @@ export class Transpiler {
     this.visit(expr);
   }
 
+  static IGNORED_ANNOTATIONS = {'CONST': true, 'ABSTRACT': true, 'IMPLEMENTS': true};
+
+  isEmptyImport(n: ts.ImportDeclaration): boolean {
+    var bindings = n.importClause.namedBindings;
+    if (bindings.kind != ts.SyntaxKind.NamedImports) return false;
+    return (<ts.NamedImports> bindings).elements.every((e) => {
+      return Transpiler.IGNORED_ANNOTATIONS.hasOwnProperty(this.ident(e.name));
+    });
+  }
+
+  filterImports(ns: ts.ImportOrExportSpecifier[]) {
```

Comment:
That's much nicer indeed. Doesn't even have to be a property, works simply as a static function (could also work as a top level function).

The dead code is a bit annoying indeed. Let's see that we get to TS soon. Worst case we'd just have to rewrite these couple of functions, which is the same as just deleting it...